src-python/lab-service/traffexam/kilda/traffexam/rest.py

```diff
@@ -133,6 +133,20 @@ def address_emmit_lldp_packet(idnr):
             'sent_packets': 1}}
 
 
+@app.route('/address/<idnr>/lacp', method='PUT')
+def address_emmit_lacp_packet(idnr):
+    address = _address_lookup(unpack_idnr(idnr))
+    payload = bottle.request.json
+    try:
+        push_entry = model.LACPPush(payload)
+        get_context().action.lacp_push(address.iface, push_entry)
+    except ValueError as e:
```

Comment:
Whay about KeyError(at least for missing keys in payload)?

Reply:
Added call to `extract_payload_fields()` method.